en/render/camera.md

```diff
@@ -1,39 +1,133 @@
 
-Suppose we are making a physics game: we have physics colliders and tiled map as the background, we have `hero` node as player character. Our camera needs to follow the hero's movement so we can see him travel across the scene.
+You can add or change groupings through **Group Manager** in the **Project Settings**, which is the corresponding mask.
 
-![Camera-1](./camera/camera-1.png)
+- zoomRatio
 
-Here we need to create a new empty node and rename it to camera, use this node as a camera, using a single node as a camera node will be more flexible. Of course, we can also directly add the Camera component to the hero node, but this camera can only at the exact position of hero, and can not implement any smooth follow effect. 
+Specifies the zoom ratio of the camera, and the larger the value, the larger the image displayed.
 
-Select the camera node in the **Node Tree** and click on the `Add Component -> Add Other Component -> Camera` button below the **Properties** to add the Camera component to the camera node.
+- clearFlags
 
-<img src="./camera/camera-2.png" style="width:50%;height:50%"></img>
+Specifies the cleanup action that needs to be made when rendering the camera.
 
-Here the Camera component adds three nodes to the `targets` property, that is, we need the camera to shoot these three nodes. And we also added a `camera-control` custom components, the role of this component is mainly to move the camera node to follow the hero node.
+![camera-2](./camera/camera-2.png)
 
-This example can be found in the `tiled` example in the [Physics example](https://github.com/2youyou2/physics-example) project.
+- backgroundColor
 
-You can also refer to the [Camera Demo](https://github.com/cocos-creator/demo-camera), contains examples of the usage of camera.
+When you specify that the camera needs to clear the color, the camera uses the set background color to clear the scene.
 
-**Note**:
+- depth
 
-When using camera, if you are using the physics system that have built-in render nodes, you will need to call the associated API to add their render nodes to the camera:
+Camera depth, used to determine the order in which the camera is rendered. The larger the value, the later the camera is rendered.
+
+- targetTexture
+
+If you set the `targetTexture`, the contents of the camera render will not be printed on the screen, but will be rendered to the `targetTexture`.
+
+If you need to do some of the screen's post-effects, you can first render the screen to `targetTexture`, and then for the `targetTexture` to do the overall processing, finally through a `sprite` to show the `targetTexture`.
+
+Please refer to the [Example](https://github.com/cocos-creator/example-cases/blob/next/assets/cases/07_render_texture/render_to_sprite.js#L31)
+for details.
+
+## Camera methods
+
+- cc.Camera.findCamera
+
+`findCamera` gets the first matching camera by finding whether the `cullingMask` of all current cameras contains a node's `group`.
```

Comment:
`findCamera` gets the first matching camera by finding whether the camera `cullingMask` contains a node's `group`.